console/console-init/ui/src/Components/AddressSpace/AddressListFilter.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from "react";
+import {
+  DropdownComponent,
+  IDropdownOption
+} from "../Common/DropdownComponent";
+import {
+  InputGroup,
+  Button,
+  TextInput,
+  ButtonVariant
+} from "@patternfly/react-core";
+import { SearchIcon } from "@patternfly/react-icons";
+
+export interface IAddressListFilterProps {
+  onSearch: () => void;
+  isFilterDropdownOpen: boolean;
```

Comment:
do we need to delegate this to the user of this component? Shouldn't it be handler by the filter component itself?